src/main/deltachat/chatlist.js

```diff
@@ -99,6 +99,8 @@ module.exports = class DCChatList extends SplitOut {
       freshMessageCounter: this._dc.getFreshMessageCount(chatId),
       isArchiveLink: chat.id === C.DC_CHAT_ID_ARCHIVED_LINK,
       contactIds,
+      isSelfTalk: chat.isSelfTalk || undefined, // only transfer if needed (json strips undefined)
+      isDeviceTalk: chat.isDeviceTalk || undefined,
```

Comment:
Why strip it out? I don't think that we need to make the json strings smaller for any performance improvements.

Reply:
I would also prefer to always have it false or true and not undefined....

Reply:
wasn't space saving the whole idea of chatlist item instead of fullchat?

Reply:
No the idea was not to fetch to many unused data (causing more database queries). Memory wise we have no issue with json fields I would assume...

Reply:
then we can do that, but I'm still skeptical how fast the electron ipc can handle much json data fast.

Reply:
Don't be skeptical, measure it :) As long as there's no measurement, I wouldn't strip stuff out.